.gitignore

```diff
@@ -19,3 +19,4 @@
 *.pem
 /saml2-web-sso.cabal
 /server.yaml
+TAGS
```

Comment:
isn't this rather something for `~/.gitignore`?  (not important enough to remote it now that it's merged, just saying for future instances of this pattern.)